libs/designer/src/lib/core/state/templates/templateSlice.ts

```diff
@@ -128,6 +139,8 @@ const loadTemplateFromGithub = async (
     return {
       workflowDefinition: (templateWorkflowDefinition as any)?.default ?? templateWorkflowDefinition,
       manifest: templateManifest,
+      workflowName: templateManifest.title,
+      kind: undefined,
```

Comment:
Shouldnt we update it to something when only one of them is supported either stateful or stateless?